qa/L0_optuna_bls_model/test.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Copyright (c) 2021-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source ../common/util.sh
+source ../common/check_analyzer_results.sh
+create_logs_dir "L0_bls_model"
+
+# Set test parameters
+MODEL_ANALYZER="$(which model-analyzer)"
```

Comment:
Could you alphabetize this or organize this somehow, where possible? There are a lot of parameters, so that would make it easier to visually find what you want.